src/argus_htmx/management/commands/tailwind_config.py

```diff
@@ -12,38 +15,107 @@
+    DEFAULT_CSS_TEMPLATE_PATH = "tailwind/styles.css"
+
+    # DEFAULT_CONFIG_TARGET = "src/argus_htmx/tailwindtheme/tailwind.config.js"
+    # DEFAULT_CSS_TARGET = "src/argus_htmx/tailwindtheme/styles.css"
+    DEFAULT_CONFIG_TARGET = "tailwind.config.js"
+    DEFAULT_CSS_TARGET = "styles.css"
 
     def handle(self, *args, **options):
-        template_loc = getattr(settings, "TAILWIND_CONFIG_TEMPLATE", self.DEFAULT_TEMPLATE_PATH)
-        target_path = getattr(settings, "TAILWIND_CONFIG_TARGET", self.DEFAULT_TARGET)
+        config_template_name = getattr(
+            settings, "TAILWIND_CONFIG_TEMPLATE", self.DEFAULT_CONFIG_TEMPLATE_PATH
+        )
+        config_target_path = pathlib.Path(
+            getattr(settings, "TAILWIND_CONFIG_TARGET", self.DEFAULT_CONFIG_TARGET)
+        )
+        css_template_name = getattr(
+            settings, "TAILWIND_CSS_TEMPLATE", self.DEFAULT_CSS_TEMPLATE_PATH
+        )
+        css_target_path = pathlib.Path(
+            getattr(settings, "TAILWIND_CSS_TARGET", self.DEFAULT_CSS_TARGET)
+        )
+        self.write_template(
+            config_template_name,
+            config_target_path,
+            context=self.get_context(target_dir=config_target_path.parent),
+            name="tailwind config",
+        )
+        self.write_template(
+            css_template_name,
+            css_target_path,
+            context=self.get_context(target_dir=css_target_path.parent),
+            name="tailwind base css",
+        )
 
-        context = {
-            "themeoverride": getattr(settings, "TAILWIND_THEME_OVERRIDE", argus_htmx_settings.TAILWIND_THEME_OVERRIDE),
+    def get_context(self, target_dir: pathlib.Path):
+        return {
+            "themeoverride": getattr(
+                settings,
+                "TAILWIND_THEME_OVERRIDE",
+                argus_htmx_settings.TAILWIND_THEME_OVERRIDE,
+            ),
             "daisyuithemes": get_themes(),
+            "projectpaths": "\n".join(
+                f"        '{d}/**/*.html'," for d in self.get_template_dirs()
+            ),
+            "cssfiles": (
+                self.make_relative(p, target_dir)
+                for p in sorted(self.get_css_files(), key=lambda p: p.stem)
+            ),
         }
-        template_path = pathlib.Path(template_loc)
-        if not template_path.is_file():
-            self.stdout.write(f"{template_loc} is not a file")
-            return
 
+    def write_template(self, template_name, target_path, context, name):
         pathlib.Path(target_path).write_text(
-            self.render_config(template_path=template_path, context=context)
+            self.render_file(template_name=template_name, context=context)
         )
 
-        self.stdout.write(f"Wrote tailwind config to '{target_path}'")
+        self.stdout.write(f"Wrote {name} to '{target_path}'")
+
+    @staticmethod
+    def render_file(template_name: str, context):
```

Comment:
render_*file* might not be the best name. You're not writing the actual file, you're rendering a *string* that will be used by something else. Django would just call this `render`, other would probably call it `dumps`, for dump string.